amundsen_application/static/js/components/common/Flag/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+// TODO: Use css-modules instead of 'import'
+import './styles.scss';
+
+interface FlagProps {
+  text: string;
+  style?: string;
+}
+
+const Flag: React.SFC<FlagProps> = ({ text, style }) => {
+  return (
+    <span className={`flag label ${style}`}>{text.toUpperCase()}</span>
+  );
+};
+
+Flag.defaultProps = {
+  text: '',
+  style: 'label-default',
```

Comment:
nit: Can we call this something other than "style"? One might assume it takes a css property instead of a className.

Reply:
I can see that. I'll go with `labelStyle`.